runtime/pack/dist/opt/termdebug/plugin/termdebug.vim

```diff
@@ -1013,7 +1050,8 @@ let s:evalFromBalloonExprResult = ''
 func s:HandleEvaluate(msg)
   let value = substitute(a:msg, '.*value="\(.*\)"', '\1', '')
   let value = substitute(value, '\\"', '"', 'g')
-  let value = substitute(value, '
', '\1', '')
+  let value = substitute(value, '
+', '\1', '')
```

Comment:
:eyes:

Reply:
looks right on my end...

Reply:
i'm guessing accepting my suggested change via github changed that CR to a NL for some reason